### ISSUE TITLE
build ?

### ISSUE THREAD

**Issue by joeblew99** [aarzilli/nucular#1]
Not usre how to build the demo.

Also the vendoring.. What are you using ? So i can also do it.

x-MacBook-Pro:_demo apple$ go build demo.go
# command-line-arguments

./demo.go:43: undefined: calcDemo
./demo.go:47: undefined: newOverviewDemo

**Comment by aarzilli** [comment]
`go build demo.go` won't work because it will just compile `demo.go`, you need to use `go build`, without arguments, in that directory.

**Comment by joeblew99** [comment]
i tried that.

```
/Users/apple/workspace/go/src/github.com/aarzilli/nucular/_demo
x-MacBook-Pro:_demo apple$ ls -al
total 120
drwxr-xr-x   6 apple  staff    204 Oct 13 15:28 .
drwxr-xr-x  25 apple  staff    850 Oct 13 15:10 ..
-rw-r--r--   1 apple  staff   1522 Oct 13 15:10 calculator.go
-rw-r--r--   1 apple  staff   5342 Oct 13 15:10 demo.go
-rw-r--r--   1 apple  staff  27642 Oct 13 15:10 overview.go
-rw-r--r--   1 apple  staff  19150 Oct 13 15:10 rob.pike.mixtape.jpg
x-MacBook-Pro:_demo apple$ go build
# github.com/aarzilli/nucular/_demo
./overview.go:456: cannot use &od.CurrentWeapon (type *int) as type int in argument to w.ComboSimple

```

my env:

```
x-MacBook-Pro:_demo apple$ go version
go version go1.7.1 darwin/amd64
x-MacBook-Pro:_demo apple$ go env
GOARCH="amd64"
GOBIN="/Users/apple/workspace/go/bin"
GOEXE=""
GOHOSTARCH="amd64"
GOHOSTOS="darwin"
GOOS="darwin"
GOPATH="/Users/apple/workspace/go"
GORACE=""
GOROOT="/usr/local/go"
GOTOOLDIR="/usr/local/go/pkg/tool/darwin_amd64"
CC="clang"
GOGCCFLAGS="-fPIC -m64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fdebug-prefix-map=/var/folders/wp/ff6sz9qs6g71jnm12nj2kbyw0000gp/T/go-build008928619=/tmp/go-build -gno-record-gcc-switches -fno-common"
CXX="clang++"
CGO_ENABLED="1"
x-MacBook-Pro:_demo apple$ 

```

**Comment by joeblew99** [comment]
fixed !

remove pointer ref:
w.ComboSimple(od.Weapons, od.CurrentWeapon, 25)

**Comment by aarzilli** [comment]
That was a problem I fixed a couple of days ago c4c0d7d24d344adeb4e713a05e5ddba05580e8c1
